webapp/src/test/resources/startup.properties

```diff
@@ -68,6 +68,7 @@ debug.retry.recorder.path=${user.dir}/target/retry
 debug.libext.feed.retention.paths=${falcon.libext}
 debug.libext.feed.replication.paths=${falcon.libext}
 debug.libext.process.paths=${falcon.libext}
+debug.extension.store.uri=jail://global:00/extension/store
```

Comment:
@sowmyaramesh We can't use {user.dir}. It will run into an error because the initialization function in AbstractTestExtensionStore has a different {user.dir} value than the one where test falcon server is running. We need a consistent uri to initialize extension store. I choose to use fail FS because we also use this in test cluster definition (see webapp/src/test/resources/cluster-template.xml).